readme.md

```diff
@@ -99,6 +99,15 @@ Properties from `options` will override properties in the parsed `url`.
 
 If no protocol is specified, it will default to `https`.
 
+##### baseUrl
+
+Type: `string` `Object`
+
+When specified, `url` will be preceded by `baseUrl`.<br>
```

Comment:
"will be _prepended_ with `baseUrl`" is a bit more clear.

Should we note that this only applies to relative URLs? If you specify an absolute URL it will skip the `baseUrl`.

Reply:
The placement in the docs has this as a separate parameter to the `got` api. I think it's just part of the options, however. If that's the case it should be moved into the options section below.

Reply:
Yes, it's an option. Done in next commit.